pkg/imagesystem/buildertemplate.go

```diff
@@ -11,189 +11,209 @@ import (
+							Env: []corev1.EnvVar{
+								{
+									Name:  "ACORN_BUILD_SERVER_UUID",
+									Value: builderUID,
+								},
+								{
+									Name:  "ACORN_BUILD_SERVER_NAMESPACE",
+									Value: forNamespace,
+								},
+								{
+									Name:  "ACORN_BUILD_SERVER_FORWARD_SERVICE",
+									Value: forwardAddress + fmt.Sprintf(":%d", system.RegistryPort),
+								},
+								{
+									Name: "ACORN_BUILD_SERVER_PUBLIC_KEY",
+									ValueFrom: &corev1.EnvVarSource{
+										SecretKeyRef: &corev1.SecretKeySelector{
+											LocalObjectReference: corev1.LocalObjectReference{
+												Name: name,
 											},
+											Key: "pub",
 										},
 									},
-									{
-										Name: "ACORN_BUILD_SERVER_PRIVATE_KEY",
-										ValueFrom: &corev1.EnvVarSource{
-											SecretKeyRef: &corev1.SecretKeySelector{
-												LocalObjectReference: corev1.LocalObjectReference{
-													Name: name,
-												},
-												Key: "priv",
+								},
+								{
+									Name: "ACORN_BUILD_SERVER_PRIVATE_KEY",
+									ValueFrom: &corev1.EnvVarSource{
+										SecretKeyRef: &corev1.SecretKeySelector{
+											LocalObjectReference: corev1.LocalObjectReference{
+												Name: name,
 											},
+											Key: "priv",
 										},
 									},
 								},
-								Args: []string{
-									"build-server",
-								},
-								ReadinessProbe: &corev1.Probe{
-									ProbeHandler: corev1.ProbeHandler{
-										TCPSocket: &corev1.TCPSocketAction{
-											Port: intstr.IntOrString{
-												IntVal: int32(system.BuildkitPort),
-											},
+							},
+							Args: []string{
+								"build-server",
+							},
+							ReadinessProbe: &corev1.Probe{
+								ProbeHandler: corev1.ProbeHandler{
+									TCPSocket: &corev1.TCPSocketAction{
+										Port: intstr.IntOrString{
+											IntVal: int32(system.BuildkitPort),
 										},
 									},
-									InitialDelaySeconds: 2,
-									PeriodSeconds:       5,
 								},
-								Ports: []corev1.ContainerPort{
-									{
-										ContainerPort: int32(system.BuildkitPort),
-									},
+								InitialDelaySeconds: 2,
+								PeriodSeconds:       5,
+							},
+							Ports: []corev1.ContainerPort{
+								{
+									ContainerPort: int32(system.BuildkitPort),
 								},
-								VolumeMounts: []corev1.VolumeMount{
-									{
-										Name:      "socket",
-										MountPath: "/run/buildkit",
-									},
+							},
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "socket",
+									MountPath: "/run/buildkit",
 								},
 							},
 						},
-						Volumes: []corev1.Volume{
-							{
-								Name: "socket",
-								VolumeSource: corev1.VolumeSource{
-									EmptyDir: &corev1.EmptyDirVolumeSource{},
-								},
+					},
+					Volumes: []corev1.Volume{
+						{
+							Name: "socket",
+							VolumeSource: corev1.VolumeSource{
+								EmptyDir: &corev1.EmptyDirVolumeSource{},
```

Comment:
Nothing really changes here, just moving it out so we can modify the object based on custom bundle.